api-docs/api-reference/methods/users/post-add-user-v2.0.rst

```diff
@@ -35,6 +35,14 @@ page in the Cloud Control panel.
    - If you try to create an account with the user name from an existing
      account, the Add user operation returns an ``HTTP error 409`` message.
 
+   - Domain multi-factor authentication, MFA, enforcement is set to
+     ``RACKSPACE_MANDATED`` for all new dedicated and cloud accounts.
+     This means that the new user-admin will need to configure MFA
```

Comment:
This means that the new user-admin must configure MFA...

Reply:
Done